src/Settings/ElevarSettings.php

```diff
@@ -8,11 +8,17 @@
 class ElevarSettings extends AppSettings
 {
     public string $uuid;
+    public bool $data_layer_listener_enabled;
+    public string $server_side_url;
 
     public function rules(): array
     {
         return [
+
              'uuid' => Rule::requiredIf($this->enabled === true),
+             'data_layer_listener_enabled' => 'boolean',
+             'server_side_url' => ['url','nullable'],
```

Comment:
Since we don't control the `server_side_url` parameter it might be wise to not force it be a URL. There could be a world where they want us to just "gtm.helixsleep.com" and then this would bork and we would need to make a code update for it. Let's trust the person inputting this valid that they know what they are entering :) 

```suggestion
             'server_side_url' => ['nullable'],
```